glide.lock

```diff
@@ -9,28 +9,99 @@ imports:
   version: 4c0e84591b9aa9e6dcfdf3e020114cd81f89d5f9
   subpackages:
   - quantile
+- name: github.com/boltdb/bolt
```

Comment:
Sounds good.  Yeah glide is pretty bad with the test imports and conflicts stuff.